client/web/compose/src/components/Chart/Report/RadarChart.vue

```diff
@@ -19,6 +19,7 @@
               />
               <b-input-group-append>
                 <chart-translator
+                  :tooltip="$t('general.translations')"
```

Comment:
Instead of doing it on every chart translator, might make sense to just do it in the chart-translator component dirrectly.